plugins/modules/clickhouse_info.py

```diff
@@ -27,6 +27,7 @@
 
 author:
   - Andrew Klychkov (@Andersson007)
+  - Aleksandr Vagachev (@aleksvagachev)
```

Comment:
```suggestion
  - Aleksandr Vagachev (@aleksvagachev)
```
Ah, was going to say that feel free to add yourself as an author even when you fix stuff (just a fyi for future PRs).